src/less/select/ds6/select.less

```diff
@@ -31,6 +31,13 @@ span.select__icon {
     }
 }
 
+[dir="rtl"] {
+    .select& > select {
```

Comment:
i know existing code, maybe a backlog task to start thinking about not having tag selectors. 
`.select& > select.className` or `.select& > .child-select` or similar.

Reply:
@PrashantAshok This helps enforce this particular component’s markup. E.g. you _must have_ a `<select>` or it won’t work correctly.

Classes will be used for the other “dropdown” types (menu and combobox) for sure.

Reply:
In other words, this is a deliberate selector.

Reply:
If we want to enforce markup, we can just append a class name to it. `select.className` works & would have advantage of faster lookup vs just `select`.

Reply:
Four year old article:

>sweating over the selectors used in modern browsers is futile; most selection methods are now so fast it’s really not worth spending much time over. Furthermore, there is disparity across browsers of what the slowest selectors are anyway. Look here last to speed up your CSS.

https://benfrain.com/css-performance-revisited-selectors-bloat-expensive-styles/

It’s not really an issue.